timeline/views_ajax.py

```diff
@@ -239,12 +228,44 @@ def get(self, request, *args, **kwargs):
             'timeline.view_classified_site_event'
         ):
             return HttpResponseForbidden()
+        return Response(self.get_event_extra(event), status=200)
 
-        if 'idx' in self.kwargs:
-            status = event.get_status_changes().reverse()
-            return Response(
-                self.get_event_extra(event, status[int(self.kwargs['idx'])]),
-                status=200,
-            )
+
+class EventStatusExtraAjaxView(
+    EventExtraDataMixin, SODARBasePermissionAjaxView
+):
+    """Ajax view for retrieving event status extra data for events"""
+
+    permission_required = 'timeline.view_status_extra'
```

Comment:
We still use this perm which only has `is_authenticated` as predicate. It should be removed completely.

Also, if we don't check perms within the class at all, only in `get()`, we should use `SODARBaseAjaxView` instead of `SODARBasePermissionAjaxView`. At that point the `permission_required` member can be omitted completely.